src/components/toast/toast.component.js

```diff
@@ -130,6 +138,8 @@ const Toast = React.forwardRef(
             {...tagComponent(restProps["data-component"] || "toast", restProps)}
             {...toastProps}
             ref={toastContentNodeRef}
+            tabIndex={tabIndex}
+            onBlur={() => setTabIndex(undefined)}
```

Comment:
really the tabIndex should be -1. That can cause issues with Cypress tests though, which was why in Dialog etc. I changed it to open with a tabIndex of 0 that is removed on blur (to prevent the container being refocusable with the keyboard).

Does setting tabIndex to -1 definitely cause issues here with Toast? If not then I'd prefer using that. If it does then we need to ensure that the `tabIndex` is only set to 0 when `disableAutoFocus` is `false`, to ensure that the user can't tab into the toast container manually. (If autofocus is disabled there is no reason to want to programatically focus the container either.)

Reply:
I'll investigate and update if I don't find any issues. Good point about the outo focus behaviour, I'll make sure the guard is added if needed